xmipp

```diff
@@ -960,9 +965,15 @@ if __name__ == '__main__':
         ok = (buildConfig.check()
             and downloadDeepLearningModels(buildDir)
             and getDependencies()
-            and getSources(branch)
-            and compile(Nproc)
+            and getSources(branch))
+        okCom, strConfigInfo = compile(Nproc)
```

Comment:
I don't like that `compile()` would return a config info. Those are two independent things.
If you want to add some reporting, do it around line 965